mkdocs.yml

```diff
@@ -0,0 +1,129 @@
+site_name: Testing with Kolena
+site_description: Python client for Kolena's machine learning (ML) testing and debugging platform
+site_url: https://docs.kolena.io
+strict: true
+repo_name: kolenaIO/kolena
+repo_url: https://github.com/kolenaIO/kolena
+copyright: Copyright &copy; 2021-2023 Kolena
+edit_uri: edit/trunk/docs/
```

Comment:
Non-blocking nit: How do we expect this to work in terms of process?

Reply:
It will prompt users to fork the repo to submit a PR. I think it's nice to include to lower the barrier to entry for small contributions (e.g. fixing spelling or grammar) and a nice prod to remind readers that the project is open-source and accepts community PRs.